makefile

```diff
@@ -7,16 +7,22 @@ local-stop:
 local-down:
 	docker-compose -f build/local/server/docker-compose.yaml down --volumes
 
+# not sure if this is good make style but it works for now
 integration-test:
-	docker-compose -f build/local/server/docker-compose.test.yaml up --build --abort-on-container-exit
-	docker-compose -f build/local/server/docker-compose.test.yaml down --volumes
-	
-integration-down:
-	docker-compose -f build/local/server/docker-compose.test.yaml down --volumes
+	export SHERLOCK_DBHOST="localhost"
+	export SHERLOCK_DBPASSWORD="password"
+	docker run --name test-postgres -e POSTGRES_PASSWORD=password -e POSTGRES_USER=sherlock -d -p 5432:5432 postgres:13
+	go test -v -race ./...
```

Comment:
Question for learning:
What's the difference between this line and line 20. I see here the arguments are different and it seems to run in a docker container. I'm don't understand why it's under unit-test as well?

Reply:
The test command itself doesn't run in docker here. It runs a postgres container that the tests can then connect to so they can perform real database operations. The main difference between this and line 20 is the use of the `-short` flag which tells the testing tool to skip any tests that require external dependencies such as postgres. 

So if you were to run `make integration-test` in this repo it would run the full test suite against a real postgres instance. If you ran `make unit-test` it would only run the unit tests (those with no external dependencies)